src/dag/dag_block_manager.cpp

```diff
@@ -0,0 +1,268 @@
+#include "dag_block_manager.hpp"
+
+namespace taraxa {
+
+DagBlockManager::DagBlockManager(addr_t node_addr, vdf_sortition::VdfConfig const &vdf_config,
+                                 optional<state_api::DPOSConfig> dpos_config, size_t capacity, unsigned num_verifiers,
+                                 std::shared_ptr<DbStorage> db, std::shared_ptr<TransactionManager> trx_mgr,
+                                 std::shared_ptr<FinalChain> final_chain, std::shared_ptr<PbftChain> pbft_chain,
+                                 logger::Logger log_time, uint32_t queue_limit)
+    : vdf_config_(vdf_config),
+      dpos_config_(dpos_config),
+      capacity_(capacity),
+      num_verifiers_(num_verifiers),
+      db_(db),
+      trx_mgr_(trx_mgr),
+      final_chain_(final_chain),
+      pbft_chain_(pbft_chain),
+      log_time_(log_time),
+      queue_limit_(queue_limit),
+      blk_status_(10000, 100),
+      seen_blocks_(10000, 100) {
```

Comment:
maybe stupid question, but why are these constants se to 1000, 100 ? Some comment or even put it in constexpr variable with comments on them might be valuable. If it is obvious and I just don't get it because of my lack of knowledge let me know... otherwise no one will have any idea what is the reason behind these constant withou comments...

Reply:
ExpirationCacheMap class defined at util.hpp. ExpirationCacheMap(uint32_t max_size, uint32_t delete_step)
There is no specific reason to set max size 10000 and delete step to 100.
I remember long time ago, we set to max size 2000 and hit issue. So change to 10000
This will depends on TPS. And the value will change depend on performance/torture testing report.